plugins/org.jboss.tools.docker.reddeer/src/org/jboss/tools/docker/reddeer/core/ui/wizards/DockerConnectionWizard.java

```diff
@@ -41,17 +43,16 @@ public DockerConnectionWizard() {
 	}
 
 	public void open() {
-		DockerExplorer de = new DockerExplorer();
-		de.open();
+		new DockerExplorerView().open();
 		new DefaultToolItem("Add Connection").click();
 		new WaitUntil(new ShellWithTextIsAvailable("New Docker Connection"));
 	}
 
 	public void finish() {
 		String shell = new DefaultShell().getText();
```

Comment:
Maybe instead of getting a title of currently active shell (there could pop up or be some modal shell between opening a wizard and finishing it) it would be better to work with a title stored in constant (you will have better maintainable code, title stored on one place). Or you could in open method initialized Shell wizardShell to a field for whole wizard and work with this shell. Choose what suit you more.